.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,31 +1,29 @@
----
-name: Bug report
-about: Create a report to identify a potential issue
-title: ''
-labels: Bug
```

Comment:
Why remove this? I liked that it automagically added the label

Reply:
It isn't removed.  Not sure why Git says it has.  If you look at the source file it has it in :D